client/src/components/Header.js

```diff
@@ -183,9 +206,17 @@ export default ({
             activeStyle={activeStyles}
             to="/feed"
           >
-            Help Board
+            {t("feed.title")}
           </NavLink>
         </li>
+        <li>
+          <Dropdown overlay={languageMenu} trigger={["click"]}>
+            <Link to="">
+              <SvgIcon src={globe} className="globe-icon-svg"></SvgIcon>
+              {languages[localization[i18n.language]].value} <DownOutlined />
```

Comment:
I would vote to remove this now. It does take up too much real estate in the header.

There are two easy alternatives:

1. My preference - just remove it
![image](https://user-images.githubusercontent.com/10546720/94280466-71ffa280-ff1b-11ea-8036-73b430a6aea5.png)
2. OR if we need to show something use the iso key instead - at the max it will be 5 chars:
![image](https://user-images.githubusercontent.com/10546720/94280554-88a5f980-ff1b-11ea-8669-a9552d3f57a8.png)
![image](https://user-images.githubusercontent.com/10546720/94280940-05d16e80-ff1c-11ea-843b-5be111bb62e9.png)

We could look into flags as an enhancement?